### DISCUSSION TITLE
Container breakpoints not working on first load chrome mobile or mobile

### DISCUSSION THREAD

**Discussion by Dannymx** [tailwindlabs/tailwindcss#10028] [Help] [answered]
I'm dealing with a strange bug with the `container` class where at certain viewport view one of the breakpoints is ignored on first load. This is happening while using Tailwind `3.2.4` and the latest version of Next.js 13.

Steps to reproduce:
- Go to [dan.mx/about](https://dan.mx/about) on chrome with the mobile view below 768px or on your mobile browser.
- You'll see on first load the container is overflowing the screen
- Navigate to `/` by *clicking* on the **dan.mx** link
- Navigate back to `/about` by *clicking* the about link
- You'll see how the `container` now is respecting the breakpoint

I've checked the output `.css` file and it contains all of the breakpoints necessary, but for some reason it skips the breakpoint it should be using on first load.

Also, for some reason this works as expected if you view this on Chrome without the mobile view, only by resizing the window.

Any ideas? Visual issue below as a gif
![breakpoint](https://user-images.githubusercontent.com/364003/206581405-f1e261d9-260f-45cb-8769-7f2dc8fda21a.gif)

**Comment by Dannymx** [comment, marked as answer]
My bad, I found the issue was related on how Next.js 13 deals with the new `head.tsx` file from the `app` directory, the meta tags are not automatically inherited by other pages.

**Comment by wongjn** [comment]
The `/about` page is missing the following [viewport meta tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag):
```html
<meta content="width=device-width, initial-scale=1" name="viewport">
```
But the homepage *does* have it. This results in when initially landing on `/about`, the viewport is actually `980px`, thus the `(min-width: 768px)` media query is used. However, when navigating to `/`, this page has the aforementioned viewport meta tag and thus adapts the viewport width to the width of the screen and thus the `(min-width: 768px)` media query no longer matches and the `(min-width: 640px)` media query is used instead. When navigating back to `/about`, the browser seems to "remember" this meta tag even though it no longer exists.

To rectify your situation, consider ensuring that the viewport meta tag appears on every page.
